src/configs/eslint/eslint.base.js

```diff
@@ -0,0 +1,70 @@
+module.exports = {
```

Comment:
We should also check our codestyle doc files. We put label `automated` to some rules. We should get actual state of eslint rule that automates a codestyle rule and whether it is actually automated